tools/commands.js

```diff
@@ -549,8 +563,8 @@ main.registerCommand({
     // match the name of the package exactly, therefore we should tell people
     // where it was created.
     Console.info(
-      packageName + ": created in ",
-      Console.path(displayPackageDir)
+      packageName + ": created in",
+      Console.path(displayPackageDir + "/")
```

Comment:
displayPackageDir comes from convertToOSPath and thus has backslashes on Windows, right? So this seems weird.